src/goto-programs/goto_contractor.h

```diff
@@ -337,34 +401,47 @@ class goto_contractort : public goto_functions_algorithm
    * Fourth, inserting assumes in the program to reflect the contracted intervals.
    * @param _goto_functions
    */
-  goto_contractort(goto_functionst &_goto_functions)
+  goto_contractort(
+    goto_functionst &_goto_functions,
+    const namespacet &ns,
+    optionst options)
```

Comment:
Please use a const-reference for passing `options` in order to not copy it every time.

Reply:
fixed.

Reply:
> fixed.

Not really. You've added `const` but not a reference, e.g. `const optionst &options`. The difference is that with a reference no copy of the `options` object is being made. Currently this unnecessary copy is being performed.

Reply:
Great, thank you.